modules/nodes-cluster-resource-configure-jdk.adoc

```diff
@@ -28,30 +28,25 @@ this documentation, and may involve setting multiple additional JVM options.
 [id="nodes-cluster-resource-configure-jdk-heap_{context}"]
 == Understanding how to override the JVM maximum heap size
 
-For many Java workloads, the JVM heap is the largest single consumer of memory.
-Currently, the OpenJDK defaults to allowing up to 1/4 (1/`-XX:MaxRAMFraction`)
-of the compute node's memory to be used for the heap, regardless of whether the
-OpenJDK is running in a container or not. It is therefore *essential* to
-override this behavior, especially if a container memory limit is also set.
-
-There are at least two ways the above can be achieved:
-
-* If the container memory limit is set and the experimental options are
-   supported by the JVM, set `-XX:+UnlockExperimentalVMOptions
-   -XX:+UseCGroupMemoryLimitForHeap`.
-+
-[NOTE]
-====
-The `UseCGroupMemoryLimitForHeap` option has been removed in JDK 11. Use `-XX:+UseContainerSupport` instead.
-====
-+
-This sets `-XX:MaxRAM` to the container memory limit, and the maximum heap size
-(`-XX:MaxHeapSize` / `-Xmx`) to 1/`-XX:MaxRAMFraction` (1/4 by default).
-
-* Directly override one of `-XX:MaxRAM`, `-XX:MaxHeapSize` or `-Xmx`.
-+
-This option involves hard-coding a value, but has the advantage of allowing a
-safety margin to be calculated.
+The default OpenJDK settings uses a maximum of 25% of the available memory for `heap` memory. This default is conservative, and it is in a `properly-configured` container environment. The default value would result in 75% of the memory assigned to an unused container. Higher default value such as 80% is more suitable for the container. 
+
+Most of the {product-title} containers replaces the OpenJDK feature for a ctar-up script with OpenJDk default. 
```

Comment:
It's not clear what this sentence means. What's a `ctar-up scrip`?

Typo: `OpenJDK`, not `OpenJDk`

Reply:
Ack.